webapp/src/components/QuickBuy/Stages/SetupStage.tsx

```diff
@@ -38,14 +38,15 @@ export const SetupStage: React.FC<StageProps> = ({ onNext, setPayment }) => {
             />
           </Grid>
 
-          <Grid xs={12} item className="pb-5">
-            <Field
-              name="email"
-              label={t('quickBuy.setup.form.email.label')}
-              variant="standard"
-              component={TextInput}
-            />
-          </Grid>
+          {/* Enable for anonymous user*/}
```

Comment:
Mamy jakiegoś taska na to?

Reply:
Nie, trzeba jeszcze ogarnąć dokładniej anonimowego uzytkownika